crypto/src/proofs/transparent.rs

```diff
@@ -422,15 +423,18 @@ impl SwapProof {
     pub fn verify(
         &self,
         anchor: tct::builder::block::Root,
-        value_commitment: value::Commitment,
+        swap_nft_value_commitment: value::Commitment,
```

Comment:
A `SwapClaim`'s effect on the transaction value balance is just to contribute the pre-committed fees, right? So if we were going to rename this from `value_commitment`, it feels like `fee_commitment` would be a better name.

Or, we could leave it as `value_commitment` to match the other actions.